src/mocks/handlers/logout.handler.ts

```diff
@@ -0,0 +1,16 @@
+import { rest } from "msw";
+import { LOGOUT_API } from "../../components/Dropdown/DropdownConstants";
+
+
+const logoutHandler = [
+    rest.get(`${LOGOUT_API}`, (_, res, ctx) => {
+        return res(
+            ctx.status(200),
+            ctx.json({
+                message: 'Signout successful',
```

Comment:
Is this the message returned by the API?

Reply:
Yes

Reply:
Can you share the API or screenshot?

Reply:
And when the API is failed?

Reply:
done

Reply:
> Can you share the API or screenshot?

sure will add it